### PR TITLE
Disable legacy eth1 related logic

### PR DESCRIPTION
**Pull request by qqqeck**
- [x] Delete Unused fields from block
  - Eth1Data
  - Deposit
  - BLSToExecution
- [x] Delete Unused fields from state
  - eth1_data
  - eth1_data_votes
  - eth1_deposit_index
  - deposit_request_start_index
- [ ] Make it work
- [ ] Clean up electra operation code according to spec
- [ ] Delete unused prefixes
- [ ] Disable POWChain
- [ ] Remove unused codes related to
  - Legacy deposit
  - Legacy withdrawal (BLSToExecutionChange)
  - Eth1Chain
- [ ] (Optional) Add execution_request field for previous(<electra) blocks and add corresponding functions
- [ ] Fix tests

### CODE REVIEW

File: proto/eth/v2/sync_committee.pb.go [hunk 1]
```diff
@@ -0,0 +1,340 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.31.0
+// 	protoc        v4.25.1
+// source: proto/eth/v2/sync_committee.proto
+
```
Comment (sb-anderson) [inline proto/eth/v2/sync_committee.pb.go:6]: sync_committee.pb.go file should be removed. run `bazel clean` and rerun scripts.

File: proto/prysm/v1alpha1/sync_committee.pb.go [hunk 2]
```diff
@@ -0,0 +1,487 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.31.0
+// 	protoc        v4.25.1
+// source: proto/prysm/v1alpha1/sync_committee.proto
+
```
Comment (sb-anderson) [inline proto/prysm/v1alpha1/sync_committee.pb.go:6]: sync_committee.pb.go file should be removed. run `bazel clean` and rerun scripts.

**Review by sb-anderson — COMMENTED** [review]
Should check BUILD.bazel and remove unused tests.

**Comment by qqqeck** [comment]
Closed due to change of plan.

**Pull request closed without merging**